composer.lock

```diff
@@ -4,7 +4,7 @@
         "Read more about it at https://getcomposer.org/doc/01-basic-usage.md#installing-dependencies",
         "This file is @generated automatically"
     ],
-    "content-hash": "7baeda382f69ca24a7267df1eaace8f1",
+    "content-hash": "ea1125c00ed02b6a9ede41b2b9f5adc8",
```

Comment:
@john-shaffer @Flynsarmy suggestions to deal with `composer.lock` files going fwd in PR's? Seems cruel to not have any warning mechanism/.gitignore then reject from PRs...

Reply:
I've reverted composer.lock but I agree, there should be some automated restriction in place so changes are rejected through PRs. My bad.

Reply:
Ahh, and now we discover what happens when this is done. PR will automatically fail github workflow because composer.lock is bad. @leonstafford @john-shaffer any ideas how we should handle this? Should we just allow changes then after merge do a composer update and commit the updated one?

Reply:
@Flynsarmy I'm not sure. Suggest we find another project that's solved this issue already and _borrow_ their technique 😸

Reply:
Thoughts: as WP2Static is not designed to run on production systems and honestly, we're not (I'm not) reviewing each package and each package's dependencies line for line as they update, it feels we're relying on other alerts as to when a 3rd party package would be compromised. 

To buffer this, if we just include the lock files in PRs and merging to master (or move to a rolling dev branch as another improvement), then give a code lock period of a fixed time before tagging releases as a way to mitigate pushing out en masse any majorly compromised releases.

Alternatively, we move away from using any 3rd party packages (outside testing) in core and replicate the functionality they provide ourselves.